analyzer/habitat/analysis/wave_scaling/resimplified.py

```diff
@@ -23,8 +24,10 @@ def resimplified_wave_scaling(
     # Check if the kernel is too "small" - if it doesn't fill a single wave
     # on the current device AND if it doesn't fill a single wave on the
     # destination device
-    if (kernel.num_blocks // origin_wave_size == 0 and
-            kernel.num_blocks // dest_wave_size == 0):
+    if (origin_wave_size == 0 or dest_wave_size == 0):
+        logger.warn(f"One or more invalid wave sizes: kernel: {kernel.name} origin: {origin_wave_size}, dest: {dest_wave_size}")
+    if ((origin_wave_size == 0 or dest_wave_size == 0) or (kernel.num_blocks // origin_wave_size == 0 and
+                                                           kernel.num_blocks // dest_wave_size == 0)):
```

Comment:
I don't think you need this much spacing here

Reply:
I think its pip8 formatting thingy, changed to single line